configure

```diff
@@ -88,6 +88,11 @@ parser.add_option("--openssl-no-asm",
     dest="openssl_no_asm",
     help="Do not build optimized assembly for OpenSSL")
 
+parser.add_option("--openssl-fips",
```

Comment:
Hmmm, even that is not consistent with the previous `add_option` calls. This shouldn't matter anyway.